ote/src/cljs/ote/views/operator_users.cljs

```diff
@@ -18,9 +18,7 @@
                  :name->label identity
                  :show-row-hover? true
                  :no-selection? true}
-    [{:name (tr [:transport-users-page :username])
```

Comment:
Do not remove user name. That translation tag is named badly. It is users full name, not username

Reply:
Doh, good catch! Fix'd.